api/v3/Address.php

```diff
@@ -42,7 +42,7 @@
  * @return array
  *   API result array
  */
-function civicrm_api3_address_create(&$params) {
```

Comment:
So this first example is a case where $params WOULD be altered as it passes through create & into fixAddress

I'm inclined to only change places where we can check that it would not (wrong as it is)

Reply:
I appreciate the conservatism & desire for safety, but to that I'd point out that

1. None of these api action functions are called directly - they are all called by the api wrapper which does *not* take its args by ref and does not care if they are altered.
2. The tests are passing :)

Reply:
OK - you are right - the api wrapper does kill any dependence...